cron/getIncrementBalance.js

```diff
@@ -0,0 +1,29 @@
+const cron = require('node-cron');
+const { familyModel } = require('../database/models');
+
+const { getLogger } = require('../helpers');
+const logger = getLogger('getIncrementBalance');
+
+async function main() {
+  try {
+    await cron.schedule('1 * * * *', async function () {
```

Comment:
This cron timing is wrong. We discussed "each 1-st day of the month" timing, don't we?